### PR TITLE
Creation of the first JavaScript project Library

### PR DESCRIPTION
**Pull request by gonjavi**
Things done in this pull request:

- Created the main Index file, adding the form for new books and create the table with titles to display the books at the bottom. 

- Added materialize for styling.

- Created the js folder in which I included the main script.js file and other files for materialize.

- Created the folder CSS in which some styling files for materialize were included.

- Completed the steps 1-6 from the oding project: https://www.theodinproject.com/courses/javascript/lessons/library

### CODE REVIEW

**Comment by gonjavi** [comment]
What to do here? Do I need modules? or any different setup?  or I just need to continue trying to fix it.

**Review by shloch — CHANGES_REQUESTED** [review]
hello @gonjavi 
Works perfect, great job..

I like the design 😊😊

## [required changes]
- Linter errors...... try to make them all green
- make sure the number of pages, accept numbers 

Waiting for last re-review hopefully

Happy coding 😊

File: js/script.js [hunk 1]
```diff
@@ -16,16 +16,22 @@ function addBookToLibrary() {
...
+  if (isNaN(pages)) {
+    msg = 'Number of pages must be a number';
     alert(msg);
-  return ok;
   }
+
+  if (isNaN(price)) {
+    msg = 'Price must be a number';
+    alert(msg);
+  }
```
Comment (shloch) [inline js/script.js]: 👏👏👏👏

**Review by shloch — APPROVED** [review]
# Hello @gonjavi 

accomplished

You can merge, happy coding

File: js/script.js (hunk 1 above)
Comment (gonjavi) [reply js/script.js]: thanks

**Pull request merged**